account_journal_report/README.rst

```diff
@@ -0,0 +1,67 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Journal Report
+==============
+
+Adds the following report in PDF and XLSX format:
+
+* Journal Ledger. It groups the moves by their entries.
+
+Configuration
+=============
+
+Users willing to access to this report should have proper Accounting & Finance rights:
+
+#. Go to *Settings / Users* and edit your user to add the corresponding access rights as follows.
+#. In *Application / Accounting & Finance*, select *Accountant* or *Adviser* options.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to *Invoicing* or *Accounting* menu
+#. Press 'Reporting > PDF Reports > Journal Ledger'
+
```

Comment:
Include a line for indicating that pressing button "Export", you'll get the report on Excel.